.idea/createivecommons.iml

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I'd prefer we keep IDE metadata directories in our `.gitignore`. E.g. there should be a `.gitignore` rule for the `.idea/` directory.